_overviews/scala3-book/taste-modeling.md

```diff
@@ -206,7 +206,7 @@ case class Person(
 val p = Person("Reginald Kenneth Dwight", "Singer")
 
 // a good default toString method
-p                // Person = Person(Reginald Kenneth Dwight,Singer)
+p                // : Person = Person(Reginald Kenneth Dwight,Singer)
```

Comment:
I’m not entirely sure what was the intent by writing “Person = Person(...)”. If the goal is to look like the output of a worksheet in Metals, then we should add the colons just before the inferred type.

Reply:
Yes, you can just show the `Person(Reginald Kenneth Dwight,Singer)` part there. 

I’m not sure why I included the `Person = Person...` part, that’s not something I normally do.